Makefile

```diff
@@ -3,7 +3,6 @@ REGISTRY ?= quay.io
 REPO ?= kubevirt
```

Comment:
The commit message does not say why and I feel this should be a different PR. Than serious upgrade of base image.